Topics/User_Experience.md

```diff
@@ -21,9 +21,18 @@ ___Established Norms___
 ___Color Theory___
 - [Colors Influence Choice](https://usabilitygeek.com/colour-user-experience-psychology/#:~:text=Colour%20plays%20a%20crucial%20role,and%20identified%20with%20your%20industry.): Users are shown certain colors to enforce actions. For example, red is commonly linked to aggressive or bad emotions and as such can be used as a cancel to dissuade people from refunding items.
 
+___Responsive Design___
+- [Responsive Design](https://devrix.com/tutorial/important-responsive-design/): Demonstrates important and benefits of responsive design, flexiblity they provide, easiblity for the consumers upon making the said software. 
+
 ___ Nielsen's 10 Usability Heuristics___
 - [Usability Heuristics](./User_Experience/Usability_Heuristics.md)
 
+### Additional UX
```

Comment:
Consider changing the “Additional UX” heading to "The 7 Universal Design Principles" to provide a more informative header. Or alternatively you can change it to “Accessibility Features of User Experience” and change “___Accessiblity Features___” to "The 7 Universal Design Principles".

Reply:
Fixed.

Reply:
I dont see the fix? More precisely, I was asking to change the "Additional UX" header because it is not very informative to the reader.

Reply:
Doing so will lead to having two "Accessibility Features" headers, any suggestions? Thanks

Reply:
I have made the change as you said in the initial feedback. Let me if that is good. Thanks.

Reply:
I don't see any changes here? Let me know if its the same for you as well.

Reply:
I have made the change for you in the latest commit. Everything looks good to me now, and it has been merged. Thanks for the contributions!